fontaine/patterns/footer-dark.php

```diff
@@ -52,11 +52,6 @@
 	<div class="wp-block-group">
```

Comment:
This block group seems unnecessary too.

With block group:
<img width="1265" alt="Screenshot 2024-08-01 at 11 41 45" src="https://github.com/user-attachments/assets/f126cddd-90f6-4535-be0f-f1acbb686e49">



Without block group (paragraph only):
<img width="1250" alt="Screenshot 2024-08-01 at 11 42 54" src="https://github.com/user-attachments/assets/65ddd225-c57d-407a-946c-815487d17646">